recipe/meta.yaml

```diff
@@ -33,7 +36,7 @@ outputs:
         - third-party-programs.txt
     build:
       script:
-        - cmake {{ cmake_args }} -DCMAKE_SKIP_INSTALL_RPATH=ON -DCMAKE_BUILD_TYPE=Release -S . -B build
+        - cmake {{ cmake_args }} -DCMAKE_REQUIRE_FIND_PACKAGE_sentencepiece=ON -DCMAKE_SKIP_INSTALL_RPATH=ON -DCMAKE_BUILD_TYPE=Release -S . -B build
```

Comment:
- `-DCMAKE_BUILD_TYPE=Release` can be dropped because of https://github.com/openvinotoolkit/openvino_tokenizers/pull/162
- `-DCMAKE_SKIP_INSTALL_RPATH=ON` can be moved to Jinja under `Unix` section, because RPATH is not valid for Windows